doc/source/train/examples/deepspeed/deepspeed_example.rst

```diff
@@ -2,7 +2,23 @@
 
 .. _deepspeed_example:
 
-DeepSpeed ZeRO-3 Distributed Training Example with Ray Train
-============================================================
+Train with DeepSpeed ZeRO-3 and Ray Train
+=========================================
+
+This is an intermediate example that shows how to do distributed training with DeepSpeed ZeRO-3 and Ray Train.
+It demonstrates how to use :ref:`Ray Dataset <data>` with DeepSpeed ZeRO-3 and Ray Train.
```

Comment:
If we're referring to the library, would use Ray Data. Otherwise, would use Ray Datasets (plural).

```suggestion
It demonstrates how to use :ref:`Ray Data <data>` with DeepSpeed ZeRO-3 and Ray Train.
```